source/common/http/http1/codec_impl.cc

```diff
@@ -376,16 +377,88 @@ void ServerConnectionImpl::onEncodeComplete() {
   }
 }
 
+void ServerConnectionImpl::handlePath(HeaderMapImpl& headers, unsigned int method) {
+  HeaderString path(Headers::get().Path);
+
+  bool is_connect = (method == HTTP_CONNECT);
+
+  // The url is relative or OPTIONS. Nothing to do here.
```

Comment:
super nitpicky: "or a wildcard for OPTIONS"?
I keep reading that the URL is options which of course it's not :-P